### PR TITLE
NXBT-3443: Add some global env var

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

File: config/jenkins.yaml [hunk 1]
```diff
@@ -22,6 +22,12 @@ jenkins:
         env:
           - key: "ARENDER_DOCKER_REGISTRY"
             value: "docker-arender.packages.nuxeo.com"
+          - key: "ARENDER_HOST_DOCKER_REGISTRY"
+            value: "docker-private-arondor-host.packages.nuxeo.com"
+          - key: "CONNECT_PROD_URL"
+            value: "https://connect.nuxeo.com/nuxeo/site"
```
Comment (troger) [inline config/jenkins.yaml]:
```suggestion
            value: "https://connect.nuxeo.com/nuxeo"
```
If we are talking about a "Connect prod URL"

File: config/jenkins.yaml [hunk 2]
```diff
@@ -22,6 +22,12 @@ jenkins:
...
+          - key: "CONNECT_PREPROD_URL"
+            value: "https://nos-preprod-connect.nuxeocloud.com/nuxeo/site"
```
Comment (troger) [inline config/jenkins.yaml]:
```suggestion
            value: "https://nos-preprod-connect.nuxeocloud.com/nuxeo"
```

**Review by ataillefer — COMMENTED** [review]
you need to ad them in the staging image too...

File: config/jenkins.yaml (hunk 1 above)
Comment (kevinleturc) [reply config/jenkins.yaml]: Replaced by `CONNECT_PROD_SITE_URL`.

**Review by ataillefer — APPROVED** [review]

**Review by troger — APPROVED** [review]

**Pull request merged**